src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -484,6 +484,11 @@ def is_saveable_module(name, value):
             sub_model = getattr(self, pipeline_component_name)
             model_cls = sub_model.__class__
 
+            # Dynamo wraps the original mode and changes the class.
+            # Is there a principled way to obtain the original class?
+            if "_dynamo.eval_frame" in str(model_cls):
+                model_cls = sub_model._orig_mod.__class__
```

Comment:
Actually think that's quite clean - another option would be the following:
```suggestion
                model_cls = sub_model.config._class_name
```
But think what you had before is actually better (maybe with a small comment)